sktm/__init__.py

```diff
@@ -312,60 +296,59 @@ def check_patchwork(self):
                 self.db.set_patchset_pending(cpw.baseurl, cpw.project_id,
                                              series.get_patch_info_list())
                 # Submit and remember a Jenkins build for the series
+                url_list = series.get_patch_url_list()
                 self.pj.append((sktm.jtype.PATCHWORK,
                                 self.jk.build(
-                                    self.jobname,
                                     baserepo=self.baserepo,
                                     ref=stablecommit,
                                     baseconfig=self.cfgurl,
                                     message_id=series.message_id,
                                     subject=series.subject,
                                     emails=series.email_addr_set,
-                                    patchwork=series.get_patch_url_list(),
+                                    patchwork=url_list,
```

Comment:
The method parameter is still named `patchwork` here (while it's changed in the `build()`, so this commit would break things)